Import/Writer/WriterInterface.php

```diff
@@ -20,13 +20,11 @@ interface WriterInterface
 {
     /**
      * Write given redirect-route to destination.
-     *
-     * @param RedirectRouteInterface $entity
      */
-    public function write(RedirectRouteInterface $entity);
+    public function write(RedirectRouteInterface $entity): void;
```

Comment:
/cc @wachterjohannes should we allow that a Interface implementation can return something? Or should we typehint this with :void? Think we should document all Interface changes in the `UPGRADE.md` file.

Reply:
i would change the interface and document it in the UPGRADE.md